cmd/kubeless/function/call.go

```diff
@@ -86,7 +96,7 @@ var callCmd = &cobra.Command{
 			req = req.AbsPath(svc.ObjectMeta.SelfLink + ":" + port + "/proxy/")
 		}
 		timestamp := time.Now().UTC()
-		req.SetHeader("event-id", fmt.Sprintf("cli-%s-%s-%s", version.VERSION, version.GITCOMMIT, timestamp.Format(time.RFC3339Nano)))
+		req.SetHeader("event-id", fmt.Sprintf("kubeless-cli-%s", getRandString(11)))
```

Comment:
why the prefix?

(Am I missing something? Is it important/useful to include the client identifier in the event id rather than some other header?)

Reply:
the function will receive only these headers (at least in the `event` parameter), that's why I am adding that prefix. It is true that there is the `event-namespace` header that includes the emitter info so this is "duplicated". Would it be meaningful/useful if we have here just a random string?

Reply:
Unlike the other event headers, I think the event-id is going to be logged/stored/hashed/indexed by *everything* that looks at these event traces.  If we can (eg) halve the number of bytes required for that, then everything becomes that tiny bit more efficient.  (In particular, naive storage sharding can become quite confused by having variation only in the suffix).